### PR TITLE
Prod standup

### PR DESCRIPTION
**Pull request by JoshAshby**

### CODE REVIEW

File: modules/tb-all-in-one/user_data.sh [hunk 1]
```diff
@@ -1,21 +1,40 @@
-#!/bin/bash
+#!/bin/bash -ex
 set -euo pipefail
 
+exec > >(tee /var/log/user-data.log|logger -t user-data -s 2>/dev/console) 2>&1
```
Comment (robacarp) [inline modules/tb-all-in-one/user_data.sh:4]: oh, it means, take everything from the rest of this script and dump it into this pipe #bash

File: modules/tb-all-in-one/user_data.sh [hunk 2]
```diff
@@ -1,21 +1,40 @@
...
 MOUNT=/mnt/${name}
 
-DEVICE_FS=`blkid -o value -s TYPE $DISK`
+echo "Checking disk format"
+# Echo is so we don't exit the script because blkid is stupid and returns
+# exit code 2 when the disk isn't formatted >:(
+DEVICE_FS=`blkid -o value -s TYPE $DISK || echo ""`
+echo "Disk is formatted? $DEVICE_FS"
 if [ "`echo -n $DEVICE_FS`" == "" ] ; then
+  echo "Formatting $DISK"
```
Comment (robacarp) [inline modules/tb-all-in-one/user_data.sh:29]: uh you're not supposed to script things like this. 

sources of fire, sources of destruction, and sources of filesystem shouldn't be automated...

**Review by robacarp — APPROVED** [review]
seems mostly harmless

except that `mkfs.ext4` command...

File: modules/tb-all-in-one/user_data.sh (hunk 2 above)
Comment (JoshAshby) [reply modules/tb-all-in-one/user_data.sh:29]: Yolo.

File: modules/tb-all-in-one/user_data.sh (hunk 1 above)
Comment (JoshAshby) [reply modules/tb-all-in-one/user_data.sh:4]: TIL

**Pull request merged**